src/Build/Construction/ProjectRootElement.cs

```diff
@@ -617,7 +617,7 @@ public int Version
         /// This can be used to see whether the file has been changed on disk
         /// by an external means.
         /// </summary>
-        public DateTime LastWriteTimeWhenRead => Link != null ? RootLink.LastWriteTimeWhenRead : _lastWriteTimeWhenRead.ToLocalTime();
+        public DateTime LastWriteTimeWhenRead => Link != null ? RootLink.LastWriteTimeWhenRead : _lastWriteTimeWhenReadUtc.ToLocalTime();
 
         internal DateTime? StreamTime = null;
```

Comment:
Should this be 
```suggestion
        internal DateTime? StreamTimeUtc = null;
```

?